File: backend/models/device.go
package models

import (
	"github.com/jinzhu/gorm"
)

type Device struct {
	gorm.Model

	Device        string
	Version       string
	Product       string
	DeviceModel   string
	Brand         string
	AndroidId     string
	SecureId      string
	SDK           string
	Width         float64
	Height        float64
	WidthLogical  float64
	HeightLogical float64
	UserId        int64
	User          User
	Logs          []Log
	ErrorLogs     []ErrorLog
}

File: core_backend/core_backend/Models/Device.cs
namespace core_backend.Models
{
    public class Device : BaseModel
    {
        public string Name { get; set; }
        public string Version { get;set; }
        public string Product { get; set; }
        public string DeviceModel { get; set; }
        public string Brand { get; set; }
        public string AndroidId { get; set; }
        public string SecureId { get; set; }
        public string SDK { get; set; }
        public float Width { get; set; }
        public float Height { get; set; }
        public float WidthLogical { get; set; }
        public float HeightLogical { get; set; }
        public string UserId { get; set; }
    }
}
